src/richie/apps/courses/models/organization.py

```diff
@@ -98,9 +98,12 @@ class OrganizationPluginModel(CMSPlugin):
     Organization plugin model handles the relation from OrganizationPlugin
     to their Organization instance
     """
-
-    organization = models.ForeignKey(Organization)
-
+    organization = models.ForeignKey(
+            "Organization",
+            related_name="organization",
```

Comment:
I think this is wrong, it should be something like "plugin_model"?

Reply:
I've edited the related name according to:
http://docs.django-cms.org/en/latest/how_to/custom_plugins.html#for-foreign-key-relations-from-other-objects